base_layer/core/src/transactions/aggregated_body.rs

```diff
@@ -117,11 +118,48 @@ impl AggregateBody {
         self.kernels.push(kernel);
     }
 
+    /// Add a kernels to the existing aggregate body
+    pub fn add_kernels(&mut self, kernels: &mut Vec<TransactionKernel>) {
```

Comment:
```suggestion
    fn add_kernels(&mut self, new_kernels: &mut Vec<TransactionKernel>) {
```

* Making the fn public could break the contract on the Block (e.g. sorting)
* Rename argument to make it clear which list is which

Reply:
Updated arguments. 
If we follow that line, should we not make add_inputs and add_outputs private as well?

Reply:
I agree. I updated this comment, but didn't hit "save" apparently.